docs/BuildFromSource.md

```diff
@@ -189,10 +223,16 @@ On macOS/Linux:
 
 By default, all of the C# projects are built. Some C# projects require NodeJS to be installed to compile JavaScript assets which are then checked in as source. If NodeJS is detected on the path, the NodeJS projects will be compiled as part of building C# projects. If NodeJS is not detected on the path, the JavaScript assets checked in previously will be used instead. To disable building NodeJS projects, specify `-noBuildNodeJS` or `--no-build-nodejs` on the command line.
 
-### Using `dotnet` on command line in this repo
+## Step 4: Make your code changes
+
+At this point, you will have all the dependencies installed and a code editor to up and running to make changes in. Once you've made changes, you will need to rebuild the project locally to pick up your changes. You'll also need to run tests locally to verify that your changes worked.
+
+The section below provides some helpful guides for using the `dotnet` CLI in the ASP.NET Core repo.
+
+## Using `dotnet` on command line in this repo
```

Comment:
Suggest moving some of this into a new part of step 3 describing building on the command line. That should introduce `.\build.cmd` and `./build.sh` as well as sourcing `.\activate.ps1` or `./activate.sh`. Note that building projects in VS Code requires starting VS Code from the command line in which you've activated the local `.dotnet/` installation.

Reply:
Moved some of the content there.

The `build.cmd` is configurable enough that it might merit its own docs page to showcase all the different combinations.